packages/loader/container-loader/src/test/attachment.spec.ts

```diff
@@ -467,7 +463,7 @@ describe("runRetriableAttachProcess", () => {
 			);
 
 			assert.strictEqual(attachmentData?.state, AttachState.Attached, "should be attached");
-			assert.notStrictEqual(attachmentData.snapshot, undefined, "should  have snapshot");
+			assert.notStrictEqual(snapshot, "should  have snapshot");
```

Comment:
Is it intentional to check the snapshot against what looks like the error message string? This happens on 489 as well.